src/components/flash/flash.js

```diff
@@ -136,6 +136,14 @@ class Flash extends React.Component {
     open: false
   }
 
+  constructor(props) {
```

Comment:
Do we still need the state object above this code if we're initializing it here now.